doc/server_API.md

```diff
@@ -120,8 +108,7 @@ On success:
 {
     "whitelists": [
         {
-            "id": 54,
-            "name": "A",
+            "unique_id": "A",
```

Comment:
There could be a lot of Modules that are bound to one whitelist - either return list of `unique_id`s or separate this to separate call - `GET /get_modules_for_whitelist/?whitelist_name=<WHITELIST_NAME>`